src/main/kotlin/solve/importer/model/ButtonModel.kt

```diff
@@ -0,0 +1,5 @@
+package solve.importer.model
+
+import javafx.scene.control.Button
+
+class ButtonModel(text: String) : Button(text)
```

Comment:
The model derives from a visual component?!

Reply:
Check out this [example](https://github.com/edvin/todomvc) how to write in the MVC approach